atflatcontrols/atflattoolbar.pas

```diff
@@ -375,9 +383,11 @@ procedure TATFlatToolbar.AddDropdown(
   else
     b.ArrowAlign:= taRightJustify;
 
-  if ADropdownEvent=nil then
+  {$ifdef FPC}
+  if ADropdownEvent = nil then
     b.OnClick:= @PopupForDropdownClick
   else
+  {$endif}
```

Comment:
so OnClick won't work in delphi? maybe add IFDEF around @ only.

Reply:
ADropDownEvent = nil causes error in delphi. First it says not enough parameters. If I add (self) to it then it says "incompatible tyoes". I looked at that for some time but finally gave up and used the conditional to move on.